plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -310,7 +370,30 @@ protected QueryRunner createQueryRunner()
         kafka.sendMessages(dateTimeFieldsProducerRecords.stream(), schemaRegistryAwareProducer(kafka));
         pinot.createSchema(getClass().getClassLoader().getResourceAsStream("date_time_fields_schema.json"), DATE_TIME_FIELDS_TABLE);
         pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("date_time_fields_realtimeSpec.json"), DATE_TIME_FIELDS_TABLE);
+    }
 
+    private void createAndPopulateJsonTable(TestingKafka kafka, TestingPinotCluster pinot)
```

Comment:
Please don't replace the order between "json table" and "json type table". It makes git diff dirty.

Reply:
I am sorry, I didn't quiet get you about the order. So, I have maintained first the creation of `createAndPopulateJsonTable` and then followed by `createAndPopulateJsonTypeTable`

Reply:
I think I got it, you meant to ensure keep `JSON_TYPE_TABLE` first and then `JSON_TYPE` to make sure the diff is proper. I chose this order thinking `JSON_TYPE_TABLE` was added after `JSON_TYPE` in the code. 

Your point does make sense. will keep it the other order